extension.json

```diff
@@ -1,9 +1,9 @@
 {
 	"name": "SemanticMediaWiki",
-	"version": "3.1.0",
+	"version": "3.1.1",
 	"author": [
-		"[https://korrekt.org Markus Krötzsch]",
-		"[https://www.entropywins.wtf/mediawiki Jeroen De Dauw]",
+		"[https://korrekt.org/ Markus Krötzsch]",
+		"[https://www.entropywins.wtf/ Jeroen De Dauw]",
```

Comment:
???

Reply:
you also have a dedicated smw page so I figured that it is best to allow people explore both of them

Reply:
The MW one has a big link to the SMW one, which is essentially a sub page. Does not matter on this branch but please don't change this elsewhere.